### DISCUSSION TITLE
iModel Local File - iModel Synchronisation failed

### DISCUSSION THREAD

**Discussion by jsoroko** [iTwin/itwinjs-core#4410] [Q&A]
Hi, 

I was trying to create iModel based on the DGN file that I saved on my computer. The DGN was exported from the current project on ProjectWise. When I click save button, the iModel appears on the list of my iModels, but after certain time it is deleted and I got the following message. 
![image](https://user-images.githubusercontent.com/68108217/193395421-0a808a69-b910-4989-b9dd-e0f536153924.png)

Anybody knows what I am doing wrong?

Thanks in advance for help.

**Comment by williamkbentley** [comment]
Hello - File upload is currently not working. We have a fix prepared and are validating it in QA. I'll reply here once file upload is operational again.

**Reply by jsoroko** [reply]
Many thanks for update @williamkbentley

**Reply by williamkbentley** [reply]
File upload workflow is working again. It now makes use of our [Synchronization API](https://developer.bentley.com/apis/synchronization/) to do the upload and conversion. Please give it a try and let me know if you have any issues.

**Reply by arnab-git-hub** [reply]
@jsoroko Can you please let us know if your workflow is working properly now? If so, can you please mark the issue as resolved.
